RenderingToolkit/IRTK_Learning_Path/README.md

```diff
@@ -0,0 +1,67 @@
+# Intel&reg; Rendering Toolkit Learning Path
+  
+## Requirements
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu 22.04
+| Software                          | Intel&reg; Rendering Toolkit (Render Kit), Jupyter Notebooks, Intel&reg; Developer Cloud (IDC)
+
+## Description
+This repo contains Jupyter Notebook Trainings for Render Kit that **has been designed to be used on the Intel Developer Cloud** for hands-on workshops.
```

Comment:
Change all references from Intel Developer Cloud to Intel® Tiber™ Developer Cloud